### PR TITLE
[Curitiba-2024] Add images and add a sponsor

### PR DESCRIPTION
**Pull request by esaito**
Adding custom images for event
Adding a sponsor

### CODE REVIEW

File: data/events/2024/curitiba/main.yml [hunk 1]
```diff
@@ -53,35 +53,41 @@ team_members: # Name is the only required field for team members.
...
     linkedin: "https://www.linkedin.com/in/pedro-alves-batistahq3"
+    image: "pedro.jpeg"
     #github: "devopsdays"
     #facebook: "https://www.facebook.com/sally.fields"
     #linkedin: 
     #website: "https://mattstratton.com"
     #image: "sally-fields.jpg"
 
-organizer_email: "curitiba@devopsdays.org" # Put your organizer email address here
+organizer_email: "devopsdayscuritiba@gmail.com" # Put your organizer email address here
```
Comment (glasnt) [inline data/events/2024/curitiba/main.yml:80]: As per [the organizers guide](https://devopsdays.org/organizing#initial-steps-for-organizing), we require that your organiser email be your city email. 

```suggestion
organizer_email: "curitiba@devopsdays.org" # Put your organizer email address here
```

Comment (esaito) [reply data/events/2024/curitiba/main.yml:80]: No one from the local (Curitiba) organization knows how to access this email: curitiba@devopsdays.org

Comment (glasnt) [reply data/events/2024/curitiba/main.yml]: Thanks for the update. If you require assistance with anything with your event, email info@ (if you used your city email, the global core organisers would also get a copy of your communications and can help more proactively).

File: data/events/2024/curitiba/main.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ location_address: "" #Optional - use the street address of your venue. This will
 nav_elements: # List of pages you want to show up in the navigation of your page.
  # - name: propose
  # - name: location
- # - name: registration
+  - name: registration
```
Comment (glasnt) [inline data/events/2024/curitiba/main.yml:40]: by enabling this, you need to add registration information

At the moment your registration page looks like this https://deploy-preview-14424--devopsdays-web.netlify.app/events/2024-curitiba/registration

File: data/events/2024/curitiba/main.yml (hunk 1 above)
Comment (phrawzty) [reply data/events/2024/curitiba/main.yml]: @esaito Hello. curitiba@devopsdays.org is a distribution list, which means that any email sent to that address is sent to the members of the list directly. There's nothing to access. Try sending an email to that list yourself and seeing what happens; the other members of your local team will see it in their inboxes.

**Pull request merged**